lib/players/ai.rb

```diff
@@ -7,7 +7,7 @@ def initialize(checker, token, min_token)
     @min_token = min_token
   end
 
-  def get_move(grid, turns_remaining)
+  def get_move(grid, turns_remaining, token)
```

Comment:
Oh, I see, it's to adhere to the same interface as the human `get_move` method. When the provided arguments aren't necessary for both scenarios there should be another way to introduce that value to the instance that needs it (maybe through dependency injection?).